rotkehlchen/rotkehlchen.py

```diff
@@ -170,6 +170,24 @@ def __init__(self, args: argparse.Namespace) -> None:
         self.shutdown_event = gevent.event.Event()
         self.migration_manager = DataMigrationManager(self)
 
+    def kill_running_tx_query_tasks(self) -> None:
```

Comment:
wtf man no? You should kill the greenlet that queries transaction for that address you are deleting. Not for all addresses.